core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt

```diff
@@ -32,19 +32,14 @@ import com.iteneum.designsystem.components.phonenumbertext.PhoneNumberTransforma
 @Composable
 fun LpOutlinedTextFieldPassword(
     modifier: Modifier,
-    value: String,
+    value: String = "",
```

Comment:
Please check in which other palces is using the value

Reply:
update, let me know if this works